iml-graphql-queries/src/snapshot.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+pub mod create {
+    use crate::Query;
+
+    pub const QUERY: &str = r#"
+            mutation CreateSnapshot($fsname: String!, $name: String!, $comment: String, $use_barrier: Boolean) {
```

Comment:
I think it's `createSnapshot` (camelCase) :)

Reply:
not for the wrapper, ref: https://graphql.org/learn/queries/#mutations